modules/op-sample-example-pipeline-tutorial.adoc

```diff
@@ -0,0 +1,49 @@
+[id='op-sample-example-pipeline-tutorial_{context}']
+= Sample example from pipeline tutorial
+
+We are using link:https://github.com/openshift/pipelines-tutorial[pipelines tutorial] as a sample example.
+
+.Procedure
+
+. Create the `TriggerBinding` resource directly from the pipelines-tutorial repository:
```

Comment:
resource from the YAML file available in the pipelines-tutorial repository:

Same suggestion for steps on line 15, 22 and 37